opam

```diff
@@ -26,6 +26,7 @@ depends: [
   "utop"         {>= "1.17"}
   "BetterErrors" {>= "0.0.1"}
   "menhir"       {>= "20160303"}
+  "re"           {>= "1.5.0"}
```

Comment:
Awesome! This will make many things easier - there's a ton of use cases for compiling to JavaScript:
If you update the npm package.json, to have a dependency on the [npm port of `re`](https://github.com/npm-ml/re) then it will continue to be installable via `npm`!